docs/source/how_tos/how_to_generate_dynamic_tags.rst

```diff
@@ -0,0 +1,96 @@
+How to Configure Dynamic Tag Generation in Picasso
+===================================================
+
+Sometimes, due to the nature of Docker image generation, it's necessary to configure image tags dynamically. This avoids the need to update the strain every time a new image is built. These tags follow a pattern previously defined by the Picasso development team. Additionally, the feature includes the ability to generate a commit and update the `config.yml` file of the repository where the Docker image was built.
+
+Configurable Pattern for Dynamic Tag Generation
+-----------------------------------------------
+
+It is important to note that this implementation **only** generates the **tag** of the Docker image dynamically. The rest of the image name is retrieved from the previously established configuration. Further below, we explain how to properly configure it to use this feature.
+
+The pattern used for generating dynamic tags is illustrated below:
+
+::
+
+    {tutor_version}-{image_tag_prefix}{timestamp}-{random_part}
+
+- ``tutor_version``: The version of Tutor used to build the image.
+- ``image_tag_prefix``: A static prefix that can be added to the image tag (string type).
+- ``timestamp``: The date and time when the image was built.
+- ``random_part``: A 4-character alphanumeric string generated randomly.
```

Comment:
instead of random part, I think it will make more sense if we refer to it as `random_suffix`

Reply:
I also think we should specify here that this suffix is also optional in this feature

Reply:
Done!